.github/dependabot.yml

```diff
@@ -12,3 +12,15 @@ updates:
     prefix: "fix"
     prefix-development: "chore"
     include: "scope"
+
+- package-ecosystem: npm
```

Comment:
I think we will have to add each new template to dependabot individually, unless we add them as packages to the top level lerna config - or add another lerna config